zeppelin-server/build.gradle

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'application'
+mainClassName = "com.nflabs.zeppelin.server.ZeppelinServer"
+applicationName = "zeppelin"
+description = "Zeppelin server"
+
+apply from: "$rootProject.projectDir/gradle/distribution.gradle"
+apply from: "$rootProject.projectDir/gradle/publish.gradle"
+apply from: "$rootProject.projectDir/gradle/copyDependency.gradle"
+  
+dependencies {
+  compile project(':zeppelin-zengine')
+  
+  compile( 'org.apache.cxf:cxf-rt-frontend-jaxrs:2.7.7') {
+    exclude(module: 'javax.ws.rs-api')
+  }
+  compile 'org.apache.cxf:cxf-rt-transports-http:2.7.7'
+  compile 'org.java-websocket:Java-WebSocket:1.3.0'
+  compile('com.wordnik:swagger-jersey-jaxrs_2.10:1.3.0') {
+    exclude(module: 'jackson-core')
+    exclude(module: 'jackson-databind')
+    exclude(module: 'jackson-annotations')
+    exclude(module: 'jackson-module-scala_2.10')
```

Comment:
If possible, it would be so cool to have Scala version extracted to a property so later migration\build woudl be possible to other versions, i.e 2.11